startup.yml

```diff
@@ -1,25 +1,21 @@
 apps:
   - name: BystroAnnotationServer
     script: bin/bystro-server.pl
```

Comment:
this should now be perl/bin/bystro-server.pl, or maybe if the installation process makes bystro-server available in PATH, that.

Reply:
Installing the perl package will place those scripts into the path.

Reply:
It's not clear what the startup or startup-dev files are used for since they are not mentioned in the repository as far as I could tell.

Reply:
Yeah, that should be clarified. startup-dev.yml is no longer used, but startup.yml is used to start the servers, use ‘make serve-dev’

Reply:
I'd vote to make fixing these yaml configuration files separate issues related to integrating the Perl with JS and other components.